app/Migrations/Version20201022154706.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace CultuurNet\UDB3\Silex\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20201022154706 extends AbstractMigration
+{
+    public function up(Schema $schema): void
+    {
+        $table = $schema->createTable('offer_popularity');
+
+        $table->addColumn(
+            'offer_id',
+            'guid',
+            [
+                'length' => 36,
+                'notnull' => true,
+            ]
+        );
+        $table->addColumn(
+            'offer_type',
+            'string',
+            [
+                'length' => 32,
+                'notnull' => true,
+            ]
+        );
+        $table->addColumn(
+            'popularity',
+            'bigint',
+            [
+                'notnull' => true,
+            ]
+        );
+        $table->addColumn(
+            'creation_date',
+            'datetime',
+            [
+                'notnull' => true,
+            ]
+        );
+
+        $table->addIndex(
+            [
+                'offer_id',
+            ],
+            'offer_id_index'
```

Comment:
Maybe we should set this as the primary key? Then it will always get an index as well.

Reply:
Then we also have a unique constraint on that column, that is not an issue for the upload?

Reply:
As noted in the meeting notes (but maybe a bit buried) the table will be truncated every time before new data is inserted.